docs/docsite/rst/scenario_guides/guide_meraki.rst

```diff
@@ -1,33 +1,34 @@
 .. _meraki_guide:
 
+******************
 Cisco Meraki Guide
-==================
+******************
 
 
 .. _meraki_guide_intro:
 
 What is Cisco Meraki?
----------------------
+=====================
 
-Cisco Meraki is an easy to use, cloud based, network infrastructure platform for enterprise environments. While most network hardware uses command line interfaces (CLIs) for configuration, Meraki uses an easy to use Dashboard hosted in the Meraki cloud. No on-premises management hardware or software is required - only the network infrastructure to run your business.
+Cisco Meraki is an easy to use, cloud-based, network infrastructure platform for enterprise environments. While most network hardware uses command-line interfaces (CLIs) for configuration, Meraki uses an easy-to-use Dashboard hosted in the Meraki cloud. No on-premises management hardware or software is required - only the network infrastructure to run your business.
```

Comment:
there's one more "easy-to-use" here, right at the beginning of this line

Reply:
Fix made on my iPad so I can’t validate but once it’s tested, it should be good.